### PR TITLE
PUBSTWO-1647 testing, bug fix & replace Arrays.asList w/List.of

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

**Review by orosz-usgs — APPROVED** [review]

File: src/main/java/gov/usgs/cida/pubs/busservice/ext/ExtPublicationContributorService.java [hunk 1]
```diff
@@ -39,34 +39,35 @@ public ExtPublicationContributorService (
...
-			for (PublicationContributor<?> contributor : publicationContributors) {
-				processPublicationContributor(contributor);
+			for (PublicationContributor<?> publicationContributor : publicationContributors) {
+				publicationContributor = processPublicationContributor(publicationContributor);
 			}
 		}
 	}
 
-	protected void processPublicationContributor(PublicationContributor<?> publicationContributor) {
+	protected PublicationContributor<?> processPublicationContributor(PublicationContributor<?> publicationContributor) {
```
Comment (mbucknell) [inline src/main/java/gov/usgs/cida/pubs/busservice/ext/ExtPublicationContributorService.java:48]: Why did you choose to return a PublicationContributor here? The parameter object's contents are still mutated.

File: src/main/java/gov/usgs/cida/pubs/busservice/ext/ExtPublicationContributorService.java [hunk 2]
```diff
@@ -39,34 +39,35 @@ public ExtPublicationContributorService (
...
 			}
 		}
+		return publicationContributor;
 	}
 
 	protected UsgsContributor processUsgsContributor(UsgsContributor contributor) {
 		UsgsContributor temp = null;
 		if (null != contributor.getOrcid()) {
-			temp = getUsgsContributorByORCID(contributor.getOrcid());
+			temp = getUsgsContributorByOrcid(contributor.getOrcid());
```
Comment (mbucknell) [inline src/main/java/gov/usgs/cida/pubs/busservice/ext/ExtPublicationContributorService.java]: Pet Peeve: I don't like variables names "temp".

**Review by mbucknell — APPROVED** [review]
Should src/main/resources/application.yml have the commented out ipds lines completely removed?

File: src/main/java/gov/usgs/cida/pubs/busservice/ext/ExtPublicationContributorService.java (hunk 1 above)
Comment (dsteinich) [reply src/main/java/gov/usgs/cida/pubs/busservice/ext/ExtPublicationContributorService.java:48]: no good answer.

File: src/main/java/gov/usgs/cida/pubs/busservice/ext/ExtPublicationContributorService.java (hunk 2 above)
Comment (dsteinich) [reply src/main/java/gov/usgs/cida/pubs/busservice/ext/ExtPublicationContributorService.java]: changed

**Pull request merged**